nengo_viz/components/sim_control.py

```diff
@@ -93,9 +93,10 @@ def get_status(self):
 
     def javascript(self):
         info = dict(uid=self.uid)
-        json = self.javascript_config(info)
-        return 'sim = new VIZ.SimControl(control, %s); ' % json + \
-               'toolbar = new VIZ.Toolbar("%s"); ' % self.viz.viz.filename 
+        fn = json.dumps(self.viz.viz.filename)
+        js = self.javascript_config(info)
+        return 'sim = new VIZ.SimControl(control, %s); ' % js + \
```

Comment:
You don't need `+` and `\` here, and it probably makes sense to have a newline. So, this:
```python
return ('sim = new VIZ.SimControl(control, %s);\n'
        'toolbar = new VIZ.Toolbar(%s);' % (js, fn))
```
This works because the Python interpreter automatically concatenates strings next to each other. And since they're in a set of brackets, they don't need a continuation character.